wgsl/extract-grammar.py

```diff
@@ -186,8 +275,10 @@ def parse(lines, i):
                         scanner_components[scanner_span.name()][last_key] = []
                 scanner_i += scanner_record_value[-1]
         if scanner_record and scanner_span.valid(scanner_lines, scanner_i):
+            # Try parsing this line
             scanner_parse = scanner_span.parse(scanner_lines, scanner_i)
             if scanner_parse[2] < 0:
+                # This line is continues the rule parsed on the immediately preceding lines.
```

Comment:
Tiny nit: This should start as "This line continues" without copula instead of "This line is continues" with copula?

Reply:
( unrelated to this change request but latest commit is not building probably because of duplicate example name "Structure declaration" @dneto0 )

Reply:
I was testing locally and committed the wrong thing.  :-)

Reply:
@dneto0 thanks for bundling the typo fix!